src/docfx/build/legacy/files/LegacyTableOfContents.cs

```diff
@@ -67,6 +67,9 @@ private sealed class LegacyTableOfContentsMetadata : LegacyTableOfContentsExperi
         {
             [JsonProperty(PropertyName = "pdf_absolute_path")]
             public string PdfAbsolutePath { get; set; }
+
+            [JsonProperty("monikers")]
+            public List<string> Monikers { get; set; }
```

Comment:
initialize to a empty list too?

Reply:
Since this is legacy, we always just set the value